understanding/21/target-size-enhanced.html

```diff
@@ -7,87 +7,103 @@
-				<li><a href="https://web.dev/accessible-tap-targets/">web.dev Accessible tap targets</a></li>
-				<li><a href="http://touchlab.mit.edu/publications/2003_009.pdf">Human Fingertips to Investigate the Mechanics of Tactile Sense (PDF)</a></li>
-				<li><a href="https://www.cs.umd.edu/hcil/trs/2006-11/2006-11.htm">One-Handed Thumb Use on Small Touchscreen Devices</a></li>
-			</ul>
-</section>
-		<section id="techniques">
-		  <h2>Techniques</h2>
-		  <p>Each numbered item in this section represents a technique or combination of techniques that the <abbr title="Web Content Accessibility Guidelines">WCAG</abbr> Working Group deems sufficient for meeting this Success Criterion. However, it is not necessary to use these particular techniques. For information on using other techniques, see <a href="understanding-techniques" class="understanding">Understanding Techniques for WCAG Success Criteria</a>, particularly the "Other Techniques" section.</p>
-			<section id="sufficient">
-			  <h3>Sufficient</h3>
-				<p class="instructions">Techniques that are sufficient to meet the Guideline or Success Criterion.</p>
-				<section class="situation">
-					<ul>
-					  <li>Ensuring that targets are at least 44 by 44 CSS pixels.</li>
-					  <li>Ensuring inline links provide sufficiently large activation target.</li>
-		          </ul>
-              </section>
-		  </section>
-			<section id="advisory">
-				<h2>Advisory</h2>
-				<p class="instructions">Although not required for conformance, the following additional techniques should be considered in order to make content more accessible. Not all techniques can be used or would be effective in all situations.</p>
-				<ul>
-				  <li>none documented</li>
-			  </ul>
-            </section>
-			<section id="failure">
-				<h2>Failure</h2>
-				<p class="instructions">The following are common mistakes that are considered failures of Success Criterion 2.5.5 by the <abbr title="Web Content Accessibility Guidelines">WCAG</abbr> Working Group.</p>
-				<ul>
-				  <li>Failure of success criterion 2.5.5 due to target being less than 44 by 44 CSS pixels.</li>
-		      </ul>
-            </section>
-</section>
-</body>
+    </section>
+  	<section id="examples">
+  	  <h2>Examples</h2>
+      <ul>      
+        <li><strong>Example 1: Buttons</strong><br> Three buttons are on-screen and the touch target area of each button is 44 by 44 CSS pixels.</li>
+        <li><strong>Example 2: Equivalent target</strong><br> Multiple targets are provided on the page that perform the same function. One of the targets is 44 by 44 CSS pixels. The other targets do not have a minimum touch target of 44 by 44 CSS pixels.</li>
+        <li><strong>Example 3: Anchor Link</strong><br> The target is an in-page link and the target is less than 44 by 44 CSS pixels. Users can scroll the screen using browser functions so target size does not need to be met.</li>
```

Comment:
> Users can scroll the screen using browser functions so target size does not need to be met.

That users can scroll to in-page links seems orthogonal to the case covered by the example. For inline linkstarget size is exempted regardless of whether scrolling is necessary. This sentence should be removed.

Reply:
I prefer not to remove this sentence. It gives a kind of explanation why in-page links need not to be 44 x 44 px. But it is a possibility to use a different wording like for example this one:

> Anchor Link
The target is an in-page link and the target is less than 44 by 44 CSS pixels. Reason: No other page need to be opened. Users can get to the destination of the link by using the tab key or by scrolling the screen.

Reply:
I'd actually say this line is quite misleading. Yes, a user can scroll ... but that's provided they know/understand where that in-page link was linking to. That's not always a given.